src/jdk.unsupported/share/classes/com/sun/nio/file/SensitivityWatchEventModifier.java

```diff
@@ -32,9 +32,17 @@
  * Defines the <em>sensitivity levels</em> when registering objects with a
  * watch service implementation that polls the file system.
  *
+ * @deprecated
+ * This {@code enum} is only used by the polling implementation of
+ * {@link java.nio.file.WatchService WatchService}.  The polling
+ * {@code WatchService} is used only on macOS and likely to be removed
+ * in a future release when a version based on the native file event
+ * notification facility becomes available.
+ *
  * @since 1.7
  */
 
+@Deprecated(since="7", forRemoval = true)
```

Comment:
I assume this should be "21" rather than "7"

Reply:
_Mea culpa_. Fixed in f00174e5f70295dff2aa2abd16559dbc81696d2d.